zest/releaser/release.py

```diff
@@ -251,9 +253,9 @@ def _release(self):
             return
 
         package = self.vcs.name
-        version = self.data['version']
+        tag = self.pypiconfig.tag_format() % ({'version': self.data['version']})
         logger.info("Doing a checkout...")
-        self.vcs.checkout_from_tag(version)
+        self.vcs.checkout_from_tag(tag)
         # ^^^ This changes directory to a temp folder.
         self.data['tagdir'] = os.path.realpath(os.getcwd())
```

Comment:
To assist possible zest.releaser extensions that want to do something with tags, it would be good to store it in the data:

```
self.data['tag'] = tag
```

Then you would need to add a line of documentation in the [`DATA`](https://github.com/zestsoftware/zest.releaser/blob/6.8.1/zest/releaser/release.py#L24) dictionary near the top of the file, something like: `'tag': "Tag we're releasing",`.